worker/uniter/modes.go

```diff
@@ -276,7 +276,7 @@ func ModeAbide(u *Uniter) (next Mode, err error) {
 
 // idleWaitTime is the time after which, if there are no uniter events,
 // the agent state becomes idle.
-var idleWaitTime = 2 * time.Second
+var idleWaitTime = 10 * time.Second
```

Comment:
Because the underlying event resolution is 5s, and I think that 5s is the *absolute* minimum; and I'd rather see units take a bit longer in yellow and then go solid green, rather than to flicker optimistically back and forth. I could maybe be argued down closer to 5s?